status_test.go

```diff
@@ -41,7 +42,10 @@ func TestStatusWithMoreThan5LinesOfLog(t *testing.T) {
 func TestStatusDetectsWipBranches(t *testing.T) {
 	output, configuration := setup(t)
 	start(configuration)
+	createFile(t, "test.txt", "contentIrrelevant")
+	next(configuration)
 	git("checkout", "master")
+	time.Sleep(2 * time.Second)
```

Comment:
ouch, do we really need that?

Reply:
I think so. On my machine i ran into the problem that it was exactly `one second ago` and the test failed. One other option would be to just assert `ago` or `second` what do you think?